perception/autoware_tensorrt_bevdet/README.md

```diff
@@ -0,0 +1,69 @@
+# tensorrt_bevdet <!-- cspell: ignore bevdet -->
+
+## Purpose
+
+tensorrt_bevdet is a dynamic 3D bev object detection package based on 6 surround view cameras.
+
```

Comment:
Consider adding a short description of BEVDet itself here, before diving into the specifics of this TensorRT implementation.  A sentence or two summarizing the algorithm's core approach would help readers unfamiliar with BEVDet understand the context.